docker/ops.go

```diff
@@ -173,8 +179,8 @@ func ProcessMessage(message *babashka.Message) (any, error) {
 			if err := json.Unmarshal([]byte(message.Args), &args); err != nil {
 				return nil, err
 			}
-
-			err := generate_sbom(message, args[0])
+			// TODO add username and password to inbound message
+			err := generate_sbom(message, args[0], "", "")
```

Comment:
@cdupuis so we need to get the creds in to here somehow?

Reply:
Yeah, but that is an area I don't understand as it deals with the incoming babushka message.

Reply:
I can have a look at this next